File: typescript/src/workshop/m3/s02_similarity/step1/ServiceFee.ts
/**
 * Krok 1: po Inline Method z ServiceFee zostało tylko wyliczenie rodzajów.
 * Usuniemy je w kroku 2, gdy wywołujący przestaną z niego korzystać.
 */
export enum Kind {
  ONLINE_BOOKING = 'ONLINE_BOOKING',
  REFUND = 'REFUND',
}

File: csharp/src/Training.Workshop/M3/S02Similarity/Step1/ServiceFee.cs
namespace Training.Workshop.M3.S02Similarity.Step1;

/// <summary>
/// Krok 1: po Inline Method z ServiceFee zostało tylko wyliczenie rodzajów.
/// Usuniemy je w kroku 2, gdy wywołujący przestaną z niego korzystać.
/// </summary>
public static class ServiceFee
{
    public enum Kind { OnlineBooking, Refund }
}
